src/modules/auth/components/MobileComponent.tsx

```diff
@@ -0,0 +1,10 @@
+import Mobile from 'src/assets/Mobile.png';
+
+export function MobileComponent(): JSX.Element {
```

Comment:
I would delete this component and use this one [mockup-mobile](https://react.daisyui.com/?path=/story/mockup-phonemockup--default) from [react-daisyui](https://www.npmjs.com/package/react-daisyui).

Reply:
I added the custom style for mockup-mobile because the dimensions did not match